pkg/bootstrap/platform/gcp.go

```diff
@@ -291,18 +291,24 @@ func zoneToRegion(z string) (string, error) {
 func (e *gcpEnv) Locality() *core.Locality {
```

Comment:
can we add some tests? we already have them for metadata() in gcp_test.go so it should be feasible